guides/NotificationOverviews.md

```diff
@@ -0,0 +1,18 @@
+# Notification Overviews
+openEQUELLA offers a variety of notifications to it's users.
+
+The following are an overview of the notifications available.
+
+## Bad URL Notifications
+Notifies a user that one of their items contains a URL that cannot be reached by openEQUELLA.  The notification will be triggered once openEQUELLA fails to contact the URL at the warning stage and at the disabled stage.  The warning threshold default is 5 tries, and can be overridden in `optional-config` via `urlChecker.triesUntilWarning`.  The disabled threshold default is 10 tries, and can be overridden in `optional-config` via `urlChecker.triesUntilDisabled`.
```

Comment:
Maybe make the retry info tabular?
```suggestion
Notifies a user that one of their items contains a URL that cannot be reached by openEQUELLA. The notification will be triggered once openEQUELLA fails to contact the URL at the warning stage and at the disabled stage.

| Threshold Type     | Default Value | Override Property                                     |
| ------------------ | ------------- | ----------------------------------------------------- |
| warning threshold  | 5 tries       | `optional-config` via `urlChecker.triesUntilWarning`  |
| disabled threshold | 10 tries      | `optional-config` via `urlChecker.triesUntilDisabled` |

```

---

#### rendered

Notifies a user that one of their items contains a URL that cannot be reached by openEQUELLA. The notification will be triggered once openEQUELLA fails to contact the URL at the warning stage and at the disabled stage.

| Threshold Type     | Default Value | Override Property                                     |
| ------------------ | ------------- | ----------------------------------------------------- |
| warning threshold  | 5 tries       | `optional-config` via `urlChecker.triesUntilWarning`  |
| disabled threshold | 10 tries      | `optional-config` via `urlChecker.triesUntilDisabled` |